src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/PlotlyWidgetViewImpl.java

```diff
@@ -126,7 +126,9 @@ private static native void _showChart(Element el, JavaScriptObject[] xyData, Str
 					height : "100%"
 				}
 			};
-			$wnd.ReactDOM.render($wnd.React.createElement(plot, props), el);
+
+            var component = $wnd.React.createElement(plot, props)
+            reactComponentDiv.@org.sagebionetworks.web.client.widget.ReactComponentDiv::render(Lorg/sagebionetworks/web/client/jsinterop/ReactNode;)(component);
```

Comment:
I was surprised to find new ugly jsni in this PR (where so much code has been converted to nice jsinterop), but this should work

Reply:
I got lazy and didn't feel like writing JsInterop code for nested objects--this was much easier